src/sptgptsd.f

```diff
@@ -1,7 +1,12 @@
 C> @file
+C> @brief Transform spectral scalar to station points.
 C>
-C> Transform spectral scalar to station points
-C> @author IREDELL @date 96-02-29
+C> PROGRAM HISTORY LOG:
+C> -  96-02-29  IREDELL
+C> - 1998-12-15  IREDELL  OPENMP DIRECTIVES INSERTED
+C> - 1999-08-18  IREDELL  OPENMP DIRECTIVE TYPO FIXED 
```

Comment:
table formatting/caps/punctuation

Reply:
@edwardhartnett I don't see any changes to sptgptsd.f? Both in terms of table fmt and capitalization of the main doc block.

Reply:
Check again? Looks good to me on my branch, and there are no outstanding commits to be pushed...